deeplake/core/dataset/deeplake_query_dataset.py

```diff
@@ -331,5 +332,10 @@ def __del__(self):
     def random_split(self, lengths: Sequence[Union[int, float]]):
```

Comment:
I think you should add an argument of `seed` as well.

Reply:
With the current PR code, are they able to control the seed with something like `deeplake.random.seed(100)` without needing it to be an argument to all the functions??

Reply:
yes this is correct